client/src/api/v3/boost.ts

```diff
@@ -1,28 +1,102 @@
 import { emit } from 'api/common/socket';
-import { BoostConfigType } from 'types/boost';
+import {
+  FileConfigT,
+  ConfigObjRT,
+  ConfigT,
+  fileConfigTypeToRuntype,
+} from 'types/boost';
+import toast from 'react-hot-toast';
+import { Runtype, Static } from 'runtypes';
+
+type payloadActionT = 'upload' | 'delete';
+
+export const configTypeToFileSuffix: { [K in ConfigT]: string } = {
```

Comment:
Do you guys prefer this to be in it's own file in the `utils` folder? Not sure how we're adding stuff to it

Reply:
I think `utils` would be nice. That directory is mainly used for functions/variables/lookup-tables that are used across files but don't necessarily belong to any file in particular.